File: 2011/20111228 - fizzbuzz - python/fizzbuzz.py

def fizzbuzz (numero):
    resultado = ''
    if eh_divisivel_por_3(numero):
        resultado = 'Fizz'
    if eh_divisivel_por_5(numero):
        resultado = resultado + 'Buzz'

    if resultado == '':
        resultado = numero
    return resultado
    
def eh_divisivel_por_3(dividendo):
    return dividendo % 3 == 0
    
def eh_divisivel_por_5(dividendo):
    return dividendo % 5 == 0
 

File: 2015/20150325 - fizzbuzz - ruby/fizzbuzz.rb
def fizzbuzz(number)  
  if number % 15 == 0
    return "FizzBuzz"
  end  

  if number % 5 == 0
    return "Buzz"
  end

  if number % 3 == 0
    return "Fizz"      
  end  

  number
end